tools/packaging/static-build/initramfs/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+from ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=America/New_York
+RUN apt-get update &&\
+    apt-get --no-install-recommends install -y software-properties-common &&\
+    add-apt-repository ppa:git-core/ppa -y &&\
```

Comment:
You shouldn't need a ppa to install git.  Or do you need it for something else?

Reply:
The git from default base ubuntu 20.04 repo don't support sparse checkout feature, during install `gen_init_cpio` tool, we will clone the kernel tree.